Backend_Application/src/main/java/dsd/cohort/application/ingredient/IngredientServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package dsd.cohort.application.ingredient;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class IngredientServiceImpl implements IngredientService {
```

Comment:
I believe the logic to create an ingredient if not found in the DB will ultimately go here. We might have a Util function to handle the actual http fetch but in this file we should have a method that creates the new ingredient object and passes it to the createIngredient method to save it to the repo

Reply:
Should we just pull the create functionality all together since ultimately it's just a copy over when we pull from the API?